sdk/servicebus/microsoft-azure-servicebus/src/main/java/com/microsoft/azure/servicebus/management/ManagementClient.java

```diff
@@ -27,14 +20,18 @@
 public class ManagementClient {
     private ManagementClientAsync asyncClient;
 
-    public ManagementClient(ConnectionStringBuilder connectionStringBuilder) {
+    public ManagementClient(ConnectionStringBuilder connectionStringBuilder) throws InterruptedException, ServiceBusException {
```

Comment:
Since this is an existing public-facing API, this will break existing customers. You could catch the checked exceptions and throw an unchecked one to main compat.